### PR TITLE
User profile features

### PR DESCRIPTION
**Pull request by memhak**
Following features have been implemented 

- Retrieve own or an arbitrary user profile information,
- Update profile,
- A new field named "Status" in user entity.

### CODE REVIEW

File: backend/src/main/java/com/eyetrade/backend/model/entity/User.java [hunk 1]
```diff
@@ -69,4 +70,9 @@
 
     @Column(name = "confirmed")
     private boolean confirmed;
+
+    @NotNull
+    @Column(name = "status")
```
Comment (gokdeemir) [inline backend/src/main/java/com/eyetrade/backend/model/entity/User.java]: What is the meaning of status? I couldn't understand

Comment (gokdeemir) [reply backend/src/main/java/com/eyetrade/backend/model/entity/User.java]: I think we can change more specific name such as privacy type

Comment (memhak) [reply backend/src/main/java/com/eyetrade/backend/model/entity/User.java]: agree, it would make more sense

**Comment by memhak** [comment]
I'm adding changes suggested by @gokdeemir, so I'm closing this request.

**Pull request closed without merging**